systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java

```diff
@@ -239,4 +240,9 @@ public static ExecutionResultData createOcUser(String username) {
         List<String> deleteCmd = Arrays.asList(CMD, "create", "user", username);
         return execute(deleteCmd, DEFAULT_SYNC_TIMEOUT, true);
     }
+
+    public static ExecutionResultData getEvents(String namespace) {
+        List<String> command = Arrays.asList(CMD, "get", "events", "-n", namespace);
+        return execute(command, ONE_MINUTE_TIMEOUT, false);
```

Comment:
why you need wait one minute for getting events?

Reply:
the past implementation also had one minute timeout

Reply:
the past implementation also had one minute timeout